src/Tribe/Main.php

```diff
@@ -3042,10 +3042,20 @@ protected function dateHelper( $date ) {
 		 * @return mixed The meta.
 		 */
 		public function getEventMeta( $id, $meta, $single = true ) {
-			$value = get_post_meta( $id, $meta, $single );
+			// Fetch Status to check what we need to do
+			$status = get_post_status( $id );
+
+			// If the post doesn't exist just bail the get_post_meta
+			if ( is_string( $status ) && 'auto-draft' !== $status ) {
+				$value = get_post_meta( $id, $meta, $single );
+			} else {
+				$value = false;
+			}
+
 			if ( $value === false ) {
-				$method = str_replace( '_Event', '', $meta );
+				$method = str_replace( array( '_Event', '_Organizer', '_Venue' ), '', $meta );
```

Comment:
There's a few duplicated `$metaTags`/`$venueTags`/`$organizerTags` that - when the prefix is stripped off of all three types - will result in identical filters below. Example:

* `_EventURL`
* `_VenueURL`
* `_OrganizerURL`

When stripped of the prefix, they all become just `URL`. Then the filter on line 3059 will be `filter_eventsDefaultURL` for all three, giving none of them distinction.

Is that intended?